gravitee-am-ui/src/app/domain/applications/creation/application-creation.component.ts

```diff
@@ -50,9 +50,7 @@ export class ApplicationCreationComponent implements OnInit {
 
     this.applicationService.create(this.application.domain, app).subscribe(data => {
       this.snackbarService.open('Application ' + data.name + ' created');
-      // needed to trick reuse route strategy, skipLocationChange to avoid /dummy to go into history
-      this.router.navigateByUrl('/dummy', { skipLocationChange: true })
-        .then(() => this.router.navigate(['/domains', this.application.domain, 'applications', data.id]));
+      this.router.navigate(['..', data.id], {relativeTo: this.route});
```

Comment:
As far as I remember, I do that to go at the top of screen after the creation.

Reply:
I created a dedicated issue in order to solve this problem globally : https://github.com/gravitee-io/issues/issues/5002